drkafka/src/main/java/com/pinterest/doctorkafka/modules/monitor/cluster/kafka/BrokerStatsMonitor.java

```diff
@@ -125,7 +134,7 @@ public void run() {
       // Backfill complete, start long-running background processor
 
       processors.clear();
-      Processor processor = new Processor(brokerStatsTopic, zkUrl, securityProtocol, consumerConfigs);
+      Processor processor = new Processor(latestOffsets, zkUrl, securityProtocol, consumerConfigs, slidingWindowSize);
```

Comment:
why do we choose to use the latest offset here?

Reply:
This is actually the last offset that the backfill processors will process. Changing the variable name.